cli/popper/commands/cmd_add.py

```diff
@@ -0,0 +1,76 @@
+import click
+import os
+import hcl
+import shutil
+import popper.utils as pu
+import popper.scm as scm
+from popper.cli import pass_context
+from distutils.dir_util import copy_tree
+
+
+@click.command('add', short_help='Import workflow from remote repo.')
+@click.argument('path', required=True)
+@click.option(
+    '--branch',
```

Comment:
for this functionality, we can use the `@<rev>` suffix so that we are consistent with the way revisions are provided in workflows. So instead of having a `--branch` flag, we would do something like

```
popper add https://github.com/org/repo/action@master
```